src/OmniSharp.Script/ScriptContextModel.cs

```diff
@@ -1,22 +1,32 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
 
 namespace OmniSharp.Script
 {
     public class ScriptContextModel
     {
-        public ScriptContextModel(string csxPath, ProjectInfo project, HashSet<string> implicitAssemblyReferences)
+        public ScriptContextModel(string csxPath, ProjectInfo project)
         {
             Path = csxPath;
-            ImplicitAssemblyReferences = implicitAssemblyReferences;
-            CommonUsings = ScriptProjectProvider.DefaultNamespaces;
+            AssemblyReferences = project.MetadataReferences.Select(a =>
+            {
+                if (a is PortableExecutableReference portableExecutableReference)
+                {
+                    return portableExecutableReference.FilePath ?? portableExecutableReference.Display;
```

Comment:
Is this ever null?

Reply:
not at the moment, but once we support plugins to meddle with this project system it might 😃